open-sauced/app
# fix: Add keyboard support to CardHorizontalBarChart and InfoTooltip

## Description

This PR fixes accessibility issues in the CardHorizontalBarChart and InfoTooltip components. It addresses the `jsx-a11y/mouse-events-have-key-events` warnings and improves keyboard navigation support.

## Related Tickets & Documents

Fixes #4133

## Mobile & Desktop Screenshots/Recordings

N/A (No visual changes, only accessibility improvements)

## Steps to QA

1. Navigate to the CardHorizontalBarChart component using a keyboard
2. Verify that you can focus on individual language bars using the Tab key
3. Press Enter or Space on a focused bar to change the description text
4. Navigate to a page containing the InfoTooltip component
5. Verify that you can open and close the tooltip using both mouse and keyboard interactions

## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?

![Accessibility](https://github.com/user-attachments/assets/17a9b4e6-ca63-4075-bd95-558c709b4bcb)
